app/api/views/alias.py

```diff
@@ -402,6 +402,9 @@ def create_contact_route(alias_id):
         return jsonify(error="request body cannot be empty"), 400
 
     user = g.user
+    if not user.is_premium():
```

Comment:
As this endpoint can be called for existing alias, I think we should only return an error when it's a new contact.

Reply:
It will be called to create a contact for existing aliases. Creating an alias is not restricted, just creating a reverse one.

Reply:
Ah I mean this endpoint can be called for existing **contact**

Reply:
Why is it called for an existing one? It should be called only to create new ones.

Reply:
Ideally it should be the case but as there are several clients out there, we can take a safer road by only rejecting the request of a new contact is about to be created, what do you think?

Reply:
Since we want to review what the clients will do on error, I'd prefer to fix the client behavior (where possible) to prevent strange code flows creeping in and having to maintain legacy behaviours in the server.

I've looked into the browser extension and android clients and none seem to be calling this route. And there are no calls to this route logged in newrelic in the last 30 days so I don't think iOS implements it either.

Reply:
This endpoint is used by ios and the browser extension to create new contact as far as i know. You can see its calls on https://onenr.io/0bRmDbovNwy

Reply:
Argh.. Do you know why I don't I see it by url? https://onenr.io/01wZK7qGbR6

Reply:
I don't know how I missed the browser creation :P I've checked it and it works as expected showing the error that currently just shows `Forbidden` we should put something like. Free users cannot create reverse alias, please upgrade.

Reply:
That's strange, I can see logs when I clicked on your link ... I checked and your Newrelic account has full access and is an admin account.